src/MoBi.Presentation/Mappers/EventAssignmentBuilderToEventAssignmentDTOMapper.cs

```diff
@@ -23,6 +23,7 @@ public EventAssignmentBuilderDTO MapFrom(EventAssignmentBuilder eventAssignmentB
          dto.ChangedEntityPath = eventAssignmentBuilder.ObjectPath == null ? string.Empty : eventAssignmentBuilder.ObjectPath.PathAsString;
          dto.NewFormula = _formulaDTOMapper.MapFrom(eventAssignmentBuilder.Formula);
          dto.UseAsValue = eventAssignmentBuilder.UseAsValue;
+         dto.Dimension = eventAssignmentBuilder.Dimension;
```

Comment:
typically not required because one points to the other but it's implementation details so why not